docs/guides/reference/ard-processing-extent.md

```diff
@@ -0,0 +1,93 @@
+* - Great Australian Bight
+  - 52HBI, 52HEK, 52HFK, 53HKD, 53HLC, 53HNA 
+* - Augusta, Western Australia
+  - 113-084 
+* - Tasman Sea
+  - 089-088, 089-091, 090-091, 55GFM 
+* - Bass Strait
+  - 54HXB 
+
+:::
+
+## Quality and metadata 
+
+The quality of ancillary inputs used in ARD generation in offshore locations is limited in comparison to the higher quality ancillary inputs used on continental Australia. Quality information for ancillary inputs is found in the proc-info (processing information) YAML metadata file within the dataset. The tier list is USER, FALLBACK, and DEFINITIVE, with the latter being the highest level of quality. Unfortunately, this information is not available to users via the datacube, so we have added an additional metadata tag to delineate data delivered in this update: `final_ancillaries = "nonstandard"`
+
+```python
+dc.find_datasets(product="ga_ls8c_ard_3", limit=10, final_ancillaries="nonstandard")
+```
+
+## Geometric quality assessment issues 
+
+[ARD](/guides/about/glossary/#ard) is a Surface Reflectance product which is derived from USGS or ESA level 1 products. The level 1 product is geometrically corrected such that the product is 'ortho-rectified' which results in enabling acquisitions from different dates to be spatially overlaid for assessment through time. The ARD product provides a comprehensive quality assurance of geometric correction known as GQA, which is found in the dataset and processing information metadata.
+
+This update features the addition of Landsat scenes and Sentinel 2 tiles over islands and reefs. Some of these Islands, such as Christmas Island or Islands South of Papua New Guinea, are at extreme high or low latitudes and are affected by persistent cloud. This presents challenges when correlating the reference image against source images, to provide statistical geometric comparison results.
+
+## BRDF quality issues 
+
+[BRDF](/guides/about/glossary/#brdf) is a solar illumination correction which adjusts reflectance intensity in differing amounts in differing directions. The light that comes from the Sun, reflects off the Earth and is received by the sensor is adjusted based on viewing/solar angle geometries of MODIS data.
+
+Sometimes over offshore locations we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which screens out unphysical BRDF values and excludes them from the solar illumination correction.
+
+## Water vapor 
+
+No ancillary source dataset difference between mainland and offshore locations.
+
+## Aerosol 
+
+A constant value of 0.06 Aerosol Optical Depth (AOD) is used as ancillary input as we do not have aerosol data for these extended locations. The constant value is based on the assumption that the average AOD data in Australian continental is around the 0.05, but islands are surrounded by the sea and impacted by the particle of salt water, so setting AOD as 0.06 is reasonable.

```

Comment:
"continental" what? Might be a missing word here

Please check my correction for "salt water particle" to make sure this makes sense

```suggestion
A constant value of 0.06 Aerosol Optical Depth (AOD) is used as ancillary input as we do not have aerosol data for these extended locations. This constant value is based on the assumption that the average AOD data in Australian continental is around the 0.05, but islands are surrounded by the sea and impacted by salt water particles, so setting AOD as 0.06 is reasonable.

```